ui/app/components/clients/charts/vertical-bar-basic.ts

```diff
@@ -4,16 +4,16 @@
  */
 
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { BAR_WIDTH, formatNumbers } from 'vault/utils/chart-helpers';
 import { formatNumber } from 'core/helpers/format-number';
-import type { SerializedChartData } from 'vault/client-counts';
-import { tracked } from '@glimmer/tracking';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
-import { format } from 'date-fns';
+
+import type { Count, MonthlyChartData } from 'vault/vault/charts/client-counts';
 
 interface Args {
-  data: SerializedChartData[];
```

Comment:
Updated to just the data relevant to the chart, not all of the data returned from the serialized `/activity` response